src/alwaysAck.js

```diff
@@ -0,0 +1,10 @@
+'use strict'
+/* @flow */
+
+import type { MessageHandler } from './index'
+
+import { ackOnComplete, ackOnError } from './ackOn'
+import parseAndHandleMessage from './parseAndHandleMessage'
+
+export default (handleError:(e:mixed) => Promise<mixed>, f:(message:Object) => Promise<mixed>): MessageHandler =>
+  parseAndHandleMessage(x => x, ackOnError(handleError), ackOnComplete, f)
```

Comment:
> For example, alwaysAck can't nack

Precisely!  Choosing the acking policy separately from the encoding policy makes much more sense to my brain.